### DISCUSSION TITLE
when will version 1 of kratos being released?

### DISCUSSION THREAD

**Discussion by ds-shashazh** [ory/kratos#2531] [Q&A] [answered]
I saw current version of Kratos is v0.10.1.

I wondered when will the version 1 launched?  And is Kratos v0.10.1 production ready?

**Comment by aeneasr** [comment]
Yes, v0.10 is a prod release

**Comment by vinckr** [comment, marked as answer]
Hello @shasha-zhang 
v.0.10 has been released, we already had another update since then, see the complete Ory changelog: https://www.ory.sh/changelog and https://github.com/ory/kratos/releases.
